bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx96-de.html

```diff
@@ -0,0 +1,103 @@
+{% block body_class %}state-fxa-default{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_96_de') }}
+{% endblock %}
+
+{% set utm_campaign = campaign + '-de' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <div class="show-fxa-supported-signed-out show-fxa-default show-fxa-not-fx show-fxa-ios show-fxa-android">
+        <div class="logo-lockup">
+          <img src="{{ static('protocol/img/logos/firefox/browser/logo.svg') }}" width="64" height="64" alt="Firefox"> <span>+</span> <img src="{{ static('protocol/img/logos/firefox/relay/logo-white.svg') }}" width="64" height="64" alt="Relay">
+        </div>
+        <h1 class="wnp-main-title">Ein Firefox Konto ist eine hervorragende Idee</h1>
+        <p class="wnp-main-tagline">
+          Mehr als nur ein Konto – mit einem Firefox Account stärkst du deine Privatsphäre nachhaltig. Profitiere von kostenlosen Tools wie Firefox Relay und schütze deine private E-Mail Adresse effektiv vor Spam und unkontrollierter Weitergabe an Dritte.
+        </p>
+
+        <p class="wnp-main-cta">
+          <a href="https://relay.firefox.com/?utm_source=mozilla.org-whatsnew96&utm_medium=referral&utm_campaign=whatsnew96-de" class="mzp-c-button mzp-t-product" data-cta-type="button" data-cta-text="Sign up for Relay">Bei Relay anmelden</a>
+        </p>
+      </div>
+
+      <div class="show-fxa-supported-signed-in">
+        <img class="wnp-main-logo" src="{{ static('protocol/img/logos/firefox/relay/logo-white.svg') }}" width="64" height="64" alt="">
+        <h1 class="wnp-main-title">Schütze deine E-Mail mit Firefox Relay</h1>
+        <p class="wnp-main-tagline">
+          E-Mail-Adressen müssen ständig im Web angegeben werden. Das ist auch OK, allerdings nur bei Seiten, denen man zu 100% vertraut. Firefox Relay - Teil deines Firefox Kontos - erstellt Aliasse, die du anstelle deiner privaten E-Mail überall da angeben kannst, wo du deine private lieber für dich behältst.
+        </p>
+
+        <p class="wnp-main-cta">
+          <a href="https://relay.firefox.com/accounts/fxa/login/?process=login" class="mzp-c-button mzp-t-product" data-cta-type="button" data-cta-text="Activate Relay in Firefox">Relay in Firefox aktivieren</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="wnp-content-extra">
+    <div class="mzp-l-content mzp-t-content-lg mzp-l-columns mzp-t-columns-three">
+      <div class="mzp-c-picto">
+        <div class="mzp-c-picto-image" aria-hidden="true">
+          {% include 'firefox/whatsnew/includes/fx96-de/relay-extension.svg' %}
+        </div>
+        <h2 class="mzp-c-picto-heading">Nutze die Erweiterung</h2>
+        <div class="mzp-c-picto-body">
+          <p>Lade dir die Erweiterung für den Firefox Browser und erstelle automatisch Aliasse, wann immer eine Webseite nach deiner E-Mail-Adresse fragt. Mit nur einem Klick.</p>
```

Comment:
**nit:** Also based on the copy, it has the wrong text. However using Google Translate (lol) this translation seems to be accurate with the English one. Maybe just update it to suit what's on the copy?